### DISCUSSION TITLE
[ALG101] LIOJ 1019：一條路走到黑，WA求解其他測資

### DISCUSSION THREAD

**Discussion by dpes8693** [Lidemy/forum#253] [其他問題]
題目: 走迷宮
https://oj.lidemy.com/problem/1019

程式碼在此: (語法是js)
https://github.com/dpes8693/LiOJ-prctice/blob/main/1019/1019.js

問題:
1.若題目input是
`
1 1
`
`
.
`
則 output 是 0 嗎?

2.想請問大家，導致 WA 的測資是什麼呢?

感謝各位~

**Comment by aszx87410** [comment]
1. 題目目前不會有這種 input
2. 你錯的測資有兩筆，其中一筆是這個，正確答案是 65：

```
18 9
...##########....#
##........###.##.#
#########.###.##.#
...#...##.###.##.#
.#.#.#.##.###.##.#
.#...#....###.##.#
.############.##.#
..............##.#
################..
```

**Reply by dpes8693** [reply]
謝謝胡立
我for in迴圈那邊寫錯了，導致counter計算錯誤
   for (x in maze) {
      for (y in maze`[x]`) {}
}

已經修正，AC了
